docs/runtime/searcher.md

```diff
@@ -41,24 +41,48 @@ directory. That way the index can be preserved between the IDE restarts.
 
 #### Suggestions Table
 
+Suggestions table stores suggestion entries.
+
 * `id` `INTEGER` - unique identifier
-* `kind` `INTEGER` - type of suggestion entry, i.e. Atom, Method, Function, or
-  Local
-* `name` `TEXT` - entry name
+* `externalId` `UUID` - external id of an IR
+* `kind` `INTEGER` - the type of suggestion entry, i.e. Atom, Method, Function,
+  or Local
+* `module` `TEXT` - the module name
+* `name` `TEXT` - the suggestion name
 * `self_type` `TEXT` - self type of the Method
 * `return_type` `TEXT` - return type of the entry
+* `scope_start` `INTEGER` - the start of the definition scope
+* `scope_end` `INTEGER` - the end of the definition scope
 * `documentation` `TEXT` - documentation string
```

Comment:
This section would be better represented as a markdown table, I think. Can you do that?